library/dash/src/main/java/com/google/android/exoplayer2/source/dash/manifest/AdaptationSet.java

```diff
@@ -71,7 +71,6 @@
    * @param accessibilityDescriptors Accessibility descriptors in the adaptation set.
    * @param essentialProperties Essential properties in the adaptation set.
    * @param supplementalProperties Supplemental properties in the adaptation set.
-   * @param producerReferenceTimeInfo Producer Reference Time Info
```

Comment:
The param is still there, it is unintended?

Reply:
No it's intended. The info will be stored in `AdaptationSet` in `DashManifest`, then extracted in `DashTimeline.getWindow()`. Then the info is accessible via `Timeline.Window.prftInfo`